charts/kube-prometheus-stack/templates/prometheus/rules-1.14/etcd.yaml

```diff
@@ -44,12 +44,12 @@ spec:
           )
         )
         > 0
-      for: {{ dig "etcdMembersDown" "for" "10m" .Values.customRules }}
+      for: {{ dig "etcdMembersDown" "for" "20m" .Values.customRules }}
       {{- with .Values.defaultRules.keepFiringFor }}
       keep_firing_for: "{{ . }}"
       {{- end }}
       labels:
-        severity: {{ dig "etcdMembersDown" "severity" "critical" .Values.customRules }}
+        severity: {{ dig "etcdMembersDown" "severity" "warning" .Values.customRules }}
```

Comment:
Should this severity change cause a `breaking change` notice?

Reply:
It is, what it is. We cant do a breaking change for all such change from upstream.